deepspeed/profiling/flops_profiler/profiler.py

```diff
@@ -0,0 +1,751 @@
+import time
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from functools import partial
+
+module_flop_count = []
+
+
+def prod(dims):
+    p = 1
+    for v in dims:
+        p *= v
+    return p
+
+
+def linear_flops_compute(input, weight, bias=None):
```

Comment:
what would be the flops for computing bias? I thought it's zero.

Reply:
You can consider the dimension of bias (like out_features), which is the number of addition. But, this will be very small probably, so that's okay if not considering that.